src/property-filter/interfaces.ts

```diff
@@ -153,8 +153,13 @@ export namespace PropertyFilterProps {
   export type Token = PropertyFilterToken;
   export type JoinOperation = PropertyFilterOperation;
   export type ComparisonOperator = PropertyFilterOperator;
-  export type FilteringProperty = PropertyFilterProperty;
   export type FilteringOption = PropertyFilterOption;
+  export interface FilteringProperty extends PropertyFilterProperty {
+    groupValuesLabel: string;
+    propertyLabel: string;
+    group?: string;
```

Comment:
These three properties are specific for property filter and not needed in the collection hooks. These properties to be removed from the collection-hooks PropertyFilterProperty type which would require the useCollection input/output types to take filtering property as generic.